src/context/AlertContext.js

```diff
@@ -0,0 +1,35 @@
+import { createContext, useContext, useState } from "react";
+import "../components/Alert/Alert.css";
+const AlertContext = createContext();
```

Comment:
Suggestion - You dont need to use Context for alert, its not neccessary, the other approach will be that while doing an api call if the api call is successful then set the alert with successful prop else with error prop

Reply:
Yes, I thought about it. Will remove it and do something about it later. But the cart functionality and wishlist functionality are working but after 1 try. Like you have to refresh and then try and it will work